docs/www/quick-start-kubernetes.md

```diff
@@ -4,91 +4,178 @@ order: 0
 ---
 # Kubernetes Quick Start Guide
 
-Redpanda is a modern streaming platform for mission critical workloads.
-Redpanda is also fully API compatible Kafka allowing you to make full
-use of the Kafka ecosystem.
+Redpanda is a modern [streaming platform](/blog/intelligent-data-api/) for mission critical workloads.
+With Redpanda you can get up and running with streaming quickly
+and be fully compatible with the [Kafka ecosystem](https://cwiki.apache.org/confluence/display/KAFKA/Ecosystem).
 
-This quick start guide to intended to help you get started with Redpanda for
-development and testing purposes. For production deployments or performance
-testing please see our [Production Deployment](/docs/production-deployment) for more information.
+This quick start guide can help you get started with Redpanda for development and testing purposes.
+For production or benchmarking, setup a [production deployment](/docs/production-deployment).
 
-Using [Helm](https://helm.sh/) is the fastest way to get started with Redpanda
-on Kubernetes. First, we need to create a cluster. There are a number of
-different ways to create a cluster, either locally or with a hosting provider.
+Using [Helm](https://helm.sh/) is the fastest way to get started with Redpanda on Kubernetes.
+To get up and running you need to create a cluster and deploy the Redpanda operator on the cluster.
 
-- [Kind](#Kind)
-- [AWS EKS](#AWS-EKS)
-- [Google GKE](#Google-GKE)
+> **_Note_** - Run a container inside the Kubernetes cluster to communicate with the Redpanda cluster.
+> Currently, a load balancer is not automatically created during deployment by default.
 
-## Using Helm to Install Redpanda
+## Prerequisites
+
+Before you start installing Redpanda you need to install the Kubernetes setup your Kubernetes environment.
```

Comment:
install the Kubernetes setup your Kubernetes environment. -> I think that should be adjusted? Does not sound right to me